conda-forge.yml

```diff
@@ -1,11 +1,12 @@
 build_platform:
+  linux_aarch64: linux_64
```

Comment:
Note that the bot has switched linux-aarch64 to cross-compilation.

Reply:
Looking a bit closer here, linux-aarch is actually completely untested due to #66. So perhaps it would be safer to undo the switch to cross-compilation (or solve #66) here, rather than hope that everything works as expected.